gnucash/report/standard-reports/net-barchart.scm

```diff
@@ -39,7 +39,6 @@
 
 ;; included since Bug726449
 (use-modules (ice-9 regex)) ;; for regexp-substitute/global, used by jpqplot
```

Comment:
Is regex still used in other parts of this file ?

Reply:
I don't think so! I think I'll remove these as a last commit